README.md

```diff
@@ -68,9 +78,9 @@ Select all that apply:
 ```
 [] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
+[x] `$(".post").html()`
 [] `document.querySelectorAll(".post")[0].innerHTML`
```

Comment:
This is a correct response: It calls innerHTML on the first returned ".post" element.